### ISSUE TITLE
Add 2 items (greyed out for now) to "Training Set Generation"

### ISSUE THREAD

**Issue by markdouglasphd** [ITISFoundation/IEC62209-service#55]
Please add the "Select VPIFA set", and "Select 2-PEAK set" selections to "Training Set Generation" in the same way that they are currently in the "Test Set Generation".
But don't add the "Modulation set" selection.

![2023-04-28 01_08_01-oSPARC - Work - Microsoft​ Edge](https://user-images.githubusercontent.com/131360748/235009445-0e0231a0-682b-42f1-9bc5-62f092816fd5.png)
